common/bcp47/timezone.xml

```diff
@@ -403,28 +403,28 @@ For terms of use, see http://www.unicode.org/copyright.html
             <type name="umawk" description="Wake Island, U.S. Minor Outlying Islands" alias="Pacific/Wake"/>
             <type name="umjon" description="Johnston Atoll, U.S. Minor Outlying Islands" deprecated="true" preferred="ushnl"/>
             <type name="ummdy" description="Midway Islands, U.S. Minor Outlying Islands" alias="Pacific/Midway"/>
-            <type name="unk" description="Unknown time zone" alias="Etc/Unknown"/>
+            <type name="unk" description="Unknown time zone" alias="Etc/Unknown Factory"/>
```

Comment:
```suggestion
            <type name="unk" description="Unknown time zone" alias="Etc/Unknown Factory" iana="Factory"/>
```

I think adding the `iana` attribute here was what we agreed in the meeting today?

Reply:
We didn't have time to finish discussing this point but I have some concerns with using `iana="Factory"`. Since ECMAScript already returns "Etc/Unknown", I think it would be a potential web compat concern if we started returning "Factory". It might be better to just change ECMA-402 to allow "Etc/Unknown" as a return value.

Reply:
If I remember correctly from the last time I investigated this, different engines handle the Etc/Unknown value returned from ICU differently. And when we discussed the prospect of adding Etc/Unknown to ECMA-402 at a TG2 meeting (I think it was June 2023), if I remember correctly Yusuke did not support adding it. Also, Anba expressed the same anti-Etc/Unknown opinion here: https://github.com/tc39/ecma402/issues/825#issuecomment-1678675723.

I'm just looking for any cross-engine consensus for what ID to return when ICU returns Etc/Unknown. I don't have a strong opinion either way.  If we can convince the other implementers that Etc/Unknown is OK, then I'd support that.